index.bs

```diff
@@ -5006,6 +5010,8 @@ must run these steps:
 <aside class=advisement>
   &#x1F6A7;
   The {{IDBTransaction/commit()}} method is new in this edition.
+  It is supported in Chrome 76, Edge 79, and Firefox 75.
```

Comment:
Firefox 74.  (https://bugzilla.mozilla.org/show_bug.cgi?id=1497007 was the tracking bug and we generally try and have the "Milestone" under the "Tracking" accordion indicate the release that the bug was fixed in.)

Reply:
Details/pre-emptive clarification is very much appreciated!